src/options.ts

```diff
@@ -19,4 +22,5 @@ export const TerminalScreenshotOptionsSchema = joi.object({
   fontFamily: joi.string().min(1).default("Monaco"),
   backgroundColor: joi.string().min(1).default("black"),
   type: joi.string().valid("png", "jpeg").default("png"),
+  theme: joi.object<ITheme>().min(1).default(""),
```

Comment:
nit: as it defaults to `undefined`, which is a supported value for the object, rather than an empty string:

```suggestion
  theme: joi.object<ITheme>().min(1),
```

Reply:
In the recent version, this was changed again from `ITheme` to a `string`. Is that intended?

The API version is completely in-memory. For example, it doesn't take the output path, but returns the produced image as a buffer.
This allows doing things like running it in tests, where setup/clean up is purely done in memory.

I suggest keeping it the same here, taking the theme object `joi.object<ITheme>()` directly instead of its path.
The CLI can still take a path and load it from there of course.